pages/add-todo.tsx

```diff
@@ -0,0 +1,157 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import { useState } from "react";
+import styles from "../styles/Home.module.css";
+import { doc } from "@firebase/firestore"; // for creating a pointer to our Document
+import { setDoc } from "firebase/firestore"; // for adding the Document to Collection
+import { firestore } from "./constants/firebase"; // firestore instance
+import SvgIcon from "@mui/material/SvgIcon";
+import {
+  Alert,
+  Box,
+  Button,
+  Container,
+  Grid,
+  ListItem,
+  ListItemText,
+  Snackbar,
+  SnackbarOrigin,
+  TextField,
+} from "@mui/material";
+
+
+import { useRouter } from 'next/router';
+import React from "react";
+
+interface State extends SnackbarOrigin {
+  open: boolean;
+}
+
+
+const AddTodo: NextPage = () => {
```

Comment:
@hashir-00 We use camel casing for naming files. So this file should be renamed to "addTodo.tsx" not "add-todo.tsx"

Reply:
Note that I have mentioned FILE NAME. NOT this specific line